stretch-assignment/digital_timer/index.js

```diff
@@ -0,0 +1,103 @@
+
+const tens = document.querySelector('#secondTens');
+const ones = document.querySelector('#secondOnes');
+const msTens = document.querySelector('#msTens');
+const msHundreds = document.querySelector('#msHundreds');
+const start = document.querySelector('#start');
+const reset = document.querySelector('#reset');
+const colon = document.querySelector('#colon');
+
+// let stopwatch = setInterval(function(){
+//   number += .01; console.log(number) }, 10);
+reset.disabled = 'true';
+let timerOff = true;
+
+///timer function for event listener
+const timer = function(){
+  let timerOn = true;
+  let tenVariable = 0;
+  let oneVariable = 0;
+  let msTenVariable = 0;
+  let msHundredVariable = 0;
+
+
+
+  start.disabled = true;
+  tens.innerHTML = 0;
+  ones.innerHTML = 0;
+  msTens.innerHTML = 0;
+  msHundreds.innerHTML = 0;
+
+/////Tens content
+if (reset.disabled === true) {
+  let tenCounter = setInterval(function(){
+     if(tens.innerHTML <= 8 && timerOn === true) {
+       tens.innerHTML = tenVariable += 1;
+       timerOn = false;
+       tens.style.color = 'red';
+       ones.style.color = 'red';
+       msTens.style.color = 'red';
+       msHundreds.style.color = 'red';
+       colon.style.color = 'red';
+       reset.disabled = false;
+       timerOff = true;
+     } else if (tens.innerHTML >= 1){
+
+
+  }}, 10000);
```

Comment:
What did you intend to do with this empty `else if` block? Also, no need for an `else if` condition here - a simple `else` statement will do.